nemo/collections/tts/losses/tacotron2loss.py

```diff
@@ -70,7 +70,7 @@ def forward(self, *, spec_pred_dec, spec_pred_postnet, gate_pred, spec_target, s
             gate_pred = torch.nn.functional.pad(gate_pred, (0, pad_amount), value=1e3)
             max_len = spec_pred_dec.shape[2]
```

Comment:
it seems `max_len` is not used anymore because of your changes. Could you remove it?